thoth/storages/graph/postgres.py

```diff
@@ -3630,6 +3630,52 @@ def get_kebechet_github_installations_active_managers(self, slug: str) -> list:
                 return active_managers
             return []
 
+    def get_kebechet_github_installation_info_for_software_environment(
```

Comment:
@pacospace should this be renamed to:
```suggestion
    def get_kebechet_github_installation_info_for_software_environment_all(
```

(to conform the naming convention)

Reply:
Yes exactly, Thanks @fridex !!